lib/twilio/audiohelper.ts

```diff
@@ -91,10 +91,17 @@ class AudioHelper extends EventEmitter {
   private _audioContext?: AudioContext;
 
   /**
-   * The AudioProcessorEventObserver instance to use
+   * The AudioProcessorEventObserver instance to use.
    */
   private _audioProcessorEventObserver: AudioProcessorEventObserver;
 
+  /**
+   * The current AudioProcessorEventObserver instance.
+   */
+  get _audioProcessorEvent(): AudioProcessorEventObserver {
+    return this._audioProcessorEventObserver;
+  }
```

Comment:
This function signature seems strange. `_audioProcessorEvent` implies that it is an `event`. I understand we cannot add a getter for _audioProcessorEventObserver since it's already a property of audio helper. How about a function that returns it instead? _getInputDevicePromise in this file. Don't forget to mark it as `internal` since we don't want to expose this in our docs.